docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support.({issue}`6576`)
+
+## Docker Image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery Connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
+
+## Hive Connector
+
+* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+  `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support reading from table in Sequencefile format that uses LZO compression. ({issue}`6452`)
+* Expose AWS HTTP Client stats via JMX. ({issue}`6503`)
+
+## Iceberg Connector
+
+* Optimize cardinality-insensitive aggregations (max, min, distinct, approx_distinct) over 
+  identity partition columns with `optimizer.optimize-metadata-queries` config property 
+  or `optimize_metadata_queries` session property. ({issue}`5199`)
+* Provide  `use_file_size_from_metadata` catalog session property and `iceberg.use-file-size-from-metadata` 
+  config property to fix query failures on tables with wrong file sizes stored in the metadata. ({issue}`6369`)
+ 
+## Kinesis Connector
+
+* Support GZIP message compression. ({issue}`6442`)
+
+## MySQL Connector
+
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
+* Improve performance for certain queries using `stddev_pop`, `stddev_samp`, `var_pop`, `var_samp` aggregation 
+  functions by pushing the aggregation and predicates computation into the remote database. ({issue}`6673`)
+
+## PostgreSQL Connector
+
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
+
+## Redshift Connector
+
+* Use dedicated Redshift JDBC driver to access Redshift. As a result, `connection-url` in catalog 
+  configuration files needs to be updated from `jdbc:postgresql:...` to `jdbc:redshift:...`. ({issue}`6465`)
+
+## SQL Server Connector
+
+* Avoid query failures due to transaction deadlocks in SQL Server by using transaction snapshot isolation. ({issue}`6274`)
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
+* Fix failure when querying tables having index and constraint. ({issue}`6464`)
```

Comment:
"an index" and "a constraint"?

what about tables with two indexes, or two constraints?

cc @Praveen2112 @kokosing

Reply:
We could have it as 
```suggestion
* Fix failure when querying tables having indexes and constraints. ({issue}`6464`)
```